README.md

```diff
@@ -181,6 +181,9 @@ charts:
     # Recommended together with a version-bumping tool like `tbump`.
     # if baseVersion is not a prerelease version (no -suffix),
     # the suffix `-0.dev` will be appended.
+    #
+    # Alternatively this can be "major", "minor", or "patch", which will set
+    # baseVersion by incrementing the latest tag in the specified component.
```

Comment:
This didn't read so clearly to me yet, here is what I think could be a refinement.

I made an attempt below.

```suggestion
    # Alternatively baseVersion can be set to "major", "minor", or "patch", then
    # baseVersion will be calculated based on the latest version tag from `git
    # describe`, but have its "major", "minor", or "patch" version increment if
    # the version isn't a pre-release.
```